tests/integration/it_utils.py

```diff
@@ -5,35 +5,85 @@
 from time import sleep
 
 import xrpl  # noqa: F401 - needed for sync tests
+from xrpl.account import get_next_valid_seq_number
+from xrpl.asyncio.account import get_next_valid_seq_number as get_seq_num_async
 from xrpl.asyncio.clients import AsyncJsonRpcClient, AsyncWebsocketClient
-from xrpl.asyncio.transaction import (
-    safe_sign_and_autofill_transaction as sign_and_autofill_async,
-)
+from xrpl.asyncio.clients.async_client import AsyncClient
 from xrpl.asyncio.transaction import (
     safe_sign_and_submit_transaction as sign_and_submit_async,
 )
-from xrpl.asyncio.transaction import (
-    send_reliable_submission as send_reliable_submission_async,
-)
 from xrpl.clients import Client, JsonRpcClient, WebsocketClient
+from xrpl.models import CustomRequest, Payment
 from xrpl.models.response import Response
 from xrpl.models.transactions.transaction import Transaction
-from xrpl.transaction import (
+from xrpl.transaction import (  # noqa: F401 - needed for sync tests
     safe_sign_and_autofill_transaction,
     safe_sign_and_submit_transaction,
-    send_reliable_submission,
 )
 from xrpl.wallet import Wallet
 
-JSON_RPC_URL = "https://s.altnet.rippletest.net:51234"
-WEBSOCKET_URL = "wss://s.altnet.rippletest.net/"
+JSON_RPC_URL = "http://127.0.0.1:5005"
+WEBSOCKET_URL = "ws://127.0.0.1:6006"
+
+JSON_TESTNET_URL = "https://s.altnet.rippletest.net:51234"
+WEBSOCKET_TESTNET_URL = "wss://s.altnet.rippletest.net"
 
 JSON_RPC_CLIENT = JsonRpcClient(JSON_RPC_URL)
 ASYNC_JSON_RPC_CLIENT = AsyncJsonRpcClient(JSON_RPC_URL)
 
 WEBSOCKET_CLIENT = WebsocketClient(WEBSOCKET_URL)
 ASYNC_WEBSOCKET_CLIENT = AsyncWebsocketClient(WEBSOCKET_URL)
 
+JSON_RPC_TESTNET_CLIENT = JsonRpcClient(JSON_TESTNET_URL)
+ASYNC_JSON_RPC_TESTNET_CLIENT = AsyncJsonRpcClient(JSON_TESTNET_URL)
+
+WEBSOCKET_TESTNET_CLIENT = WebsocketClient(WEBSOCKET_TESTNET_URL)
+ASYNC_WEBSOCKET_TESTNET_CLIENT = AsyncWebsocketClient(WEBSOCKET_TESTNET_URL)
+
+# (is_async, is_json, is_testnet) -> client
+_CLIENTS = {
+    (True, True, True): ASYNC_JSON_RPC_TESTNET_CLIENT,
+    (True, True, False): ASYNC_JSON_RPC_CLIENT,
+    (True, False, True): ASYNC_WEBSOCKET_TESTNET_CLIENT,
+    (True, False, False): ASYNC_WEBSOCKET_CLIENT,
+    (False, True, True): JSON_RPC_TESTNET_CLIENT,
+    (False, True, False): JSON_RPC_CLIENT,
+    (False, False, True): WEBSOCKET_TESTNET_CLIENT,
+    (False, False, False): WEBSOCKET_CLIENT,
+}
+
+MASTER_ACCOUNT = "rHb9CJAWyB4rj91VRWn96DkukG4bwdtyTh"
+MASTER_SECRET = "snoPBrXtMeMyMHUVTgbuqAfg1SUTb"
+MASTER_WALLET = Wallet(MASTER_SECRET, 0)
+FUNDING_AMOUNT = "1200000000"
```

Comment:
This changed from what it used to be (1000000000 w/9 zeros), was that intentional?)

Reply:
A test was failing due to lack of funds, so I just increased the number of funds.